### PR TITLE
Tg 13 tasks

### PR DESCRIPTION
**Pull request by ProDIGGY-vinilos**

### CODE REVIEW

File: src/controllers/tasks.js [hunk 1]
```diff
@@ -1,112 +1,45 @@
...
+    );
+    return res.status(200).json({
+      message: `Task with id ${id} updated succesfully`,
+      data: updatedTask,
+      error: false,
+    });
+  } catch (error) {
+    return res.status(404).json({
+      message: `Error editing the Task ${error} `,
+      error: true,
```
Comment (agustin-schiaffino) [inline src/controllers/tasks.js:19]: please remember to change the status code to 500 on every catch, and also you are missing to add data: undefined as the team agreed on

File: src/controllers/tasks.js [hunk 2]
```diff
@@ -1,112 +1,45 @@
...
+      error: false,
     });
-  }
-  if (taskReq.taskDescription === undefined || taskReq.taskDescription === '') {
-    return res.status(404).json({
-      message: 'The task must have a description',
+  } catch (error) {
+    return res.status(400).json({
+      message: `Cannot delete Task: ${error}`,
+      error: true,
```
Comment (agustin-schiaffino) [inline src/controllers/tasks.js:37]: same comment as on line 17 to 19

Comment (ProDIGGY-vinilos) [reply src/controllers/tasks.js]: Fixed

File: src/controllers/tasks.js (hunk 1 above)
Comment (ProDIGGY-vinilos) [reply src/controllers/tasks.js]: Fixed

**Comment by agustin-schiaffino** [comment]
please remeber to mark the comment as resolved once you fix them

File: src/models/Tasks.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+const taskSchema = new Schema({
+  description: { type: String },
```
Comment (agustin-schiaffino) [inline src/models/Tasks.js:6]: same comment that on the other PR, you should add required: true to the field

**Review by agustin-schiaffino — DISMISSED** [review]

**Review by agustin-schiaffino — APPROVED** [review]

**Pull request merged**